modules/account-lib/src/coin/cspr/transactionBuilderFactory.ts

```diff
@@ -25,10 +25,17 @@ export class TransactionBuilderFactory extends BaseTransactionBuilderFactory {
   }
 
   /** @inheritDoc */
-  from(raw: DeployUtil.Deploy): TransactionBuilder {
+  from(raw: string): TransactionBuilder {
     this.validateRawTransaction(raw);
     const tx = new Transaction(this._coinConfig);
-    tx.casperTx = raw;
+    const deployJson = JSON.parse(raw);
+    const deployObject = DeployUtil.deployFromJson(deployJson);
+
+    if (!deployObject) {
+      throw new InvalidTransactionError('Invalid transaction: ' + deployObject);
```

Comment:
Can you create a const INVALID_TRANSACTION const

Reply:
This is not required on source files.
We only use const for errors in test files.